.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,3 @@
+name: "CodeQL config"
+paths-ignore:
```

Comment:
is it ok for now to keep this options inside the same job?

Reply:
What do you mean? If you mean not a separate file, I don't think that's possible.

Reply:
Found this too.
https://github.blog/changelog/2022-08-31-code-scanning-customize-your-codeql-analysis-using-query-filters/


Anyway, it seems fine to go

Reply:
Yeah, that link shows that it's not possible to exclude files/folders without a config file.